lib/psci/aarch32/psci_helpers.S

```diff
@@ -91,28 +91,6 @@ func psci_do_pwrup_cache_maintenance
 	stcopr	r0, SCTLR
 	isb
 
-#if PLAT_XLAT_TABLES_DYNAMIC
```

Comment:
minor: The 264410306381 in the commit message doesn't get displayed as a link in GitHub

Reply:
I think GitHub doesn't like this system, but it's the one that checkpatch.pl likes. It's a problem of the UI anyway.